tests/e2e/specs/backend/product-query/advanced-filters.test.ts

```diff
@@ -0,0 +1,160 @@
+/**
+ * External dependencies
+ */
+import {
+	saveOrPublish,
+	selectBlockByName,
+	findToolsPanelWithTitle,
+	getFixtureProductsData,
+	shopper,
+	getToggleIdByLabel,
+} from '@woocommerce/blocks-test-utils';
+import { ElementHandle } from 'puppeteer';
+import {
+	setCheckbox,
+	unsetCheckbox,
+	ensureSidebarOpened,
```

Comment:
Is there a particular reason for us to be importing `ensureSidebarOpened` from `'@woocommerce/e2e-utils'`? All tests are failing locally as a consequence of an incorrect selector for the editor when calling `ensureSidebarOpened`. Importing `ensureSidebarOpened` from `'@wordpress/e2e-test-utils'` instead (as we already do for other tests here, such as `filter-products-by-attribute.test.ts`) solves the problem:

`import { ensureSidebarOpened } from '@wordpress/e2e-test-utils';`

Reply:
> Is there a particular reason for us to be importing `ensureSidebarOpened` from `'@woocommerce/e2e-utils'` over here while at the same time importing the same `ensureSidebarOpened` from `'@wordpress/e2e-test-utils'` for the Popular Filter [on common](https://github.com/woocommerce/woocommerce-blocks/blob/3638cd21b16647b5d0045da154949f914b5f5ea1/tests/e2e/specs/backend/product-query/common.ts#L18)?

Great catch, it's my mistake, we should use `ensureSidebarOpened` from `@wordpress/e2e-test-utils`. Updated.